src/components/AnimatedSectionList.js

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { SectionList } from 'react-native';
+import createAnimatedComponent from '../createAnimatedComponent';
+
+const SectionListWithEventThrottle = React.forwardRef((props, ref) => (
+    <SectionList scrollEventThrottle={0.0001} {...props} ref={ref} />
```

Comment:
Why do you set scrollEventThrottle to such value? Why exactly that value?

Reply:
It is exactly the same as react-native's SectionList Animated component and I remember seeing some bug without it, just can't pop into my mind which it was ))